worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -1182,6 +1182,16 @@ public int size() {
         return getBlockChangeCount();
     }
 
+    @Override
+    public BlockVector3 getMinimumPoint() {
+        return getWorld().getMinimumPoint();
+    }
+
+    @Override
+    public BlockVector3 getMaximumPoint() {
+        return getWorld().getMaximumPoint();
+    }
+
```

Comment:
Missing annotation, see one line below where it starts.

Reply:
Which annotation? The methods exist in WE EditSession